workers/ohsome_quality_analyst/indicators/mapping_saturation/ssdoubles.R

```diff
@@ -0,0 +1,41 @@
+#############################
+## Self-starting two-steps-sigmoidal function
+## TODO: Reference Authors
```

Comment:
this is the function implement by Josephine? I think it's good to have it here in R.

Reply:
Yes, this is from Josephine. I added her name as author to this R-file.